test/train-sets/same_model_test.1.dat

```diff
@@ -1,2 +1,2 @@
-1.0 |a short |b good
--1.0 |a median
+1.0 |a short small |b good
```

Comment:
why did you change this file?

Reply:
this was need to get the action/event sums the same now that they are stored per model